frontend/src/concepts/pipelines/content/configurePipelinesServer/ConfigurePipelinesServerModal.tsx

```diff
@@ -22,14 +19,13 @@ type ConfigurePipelinesServerModalProps = {
 
 const FORM_DEFAULTS: PipelineServerConfigType = {
   database: { useDefault: true, value: EMPTY_DATABASE_CONNECTION },
-  objectStorage: { newValue: EMPTY_AWS_PIPELINE_DATA },
+  objectStorage: { newValue: EMPTY_DATABASE_CONNECTION },
```

Comment:
This _feels_ wrong by the name... I'll need to come back to this.

Reply:
Yup, as noted on slack, this needs to be "undone" -- and likely the constant moved away from the file it is in before this change (as that I think is to be deleted)